Ontology/Capability/State.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Are we also adding a property as enum for all states listed in the same excel?

Reply:
Don't really understand the question, but my understanding is that we're still in a bit of flux re. the design of the state classes, so this is mostly a placeholder for now.